jetty-hazelcast/src/main/config/modules/session-store-hazelcast-embedded.mod

```diff
@@ -0,0 +1,26 @@
+#
```

Comment:
I think I'd prefer not to confuse users by bringing in 9.4 terminology like session-store, so can you rename the module to maybe just hazelcast-embedded-sessions.mod or something?